packages/core/src/components/way-autosuggest/way-autosuggest.scss

```diff
@@ -0,0 +1,115 @@
+:host {
+  font-family: var(--way-font-family, inherit);
+  font-weight: var(--way-font-weight-semibold);
+  font-size: var(--way-button-font-size-medium);
+  font-kerning: none;
+
+  --border-radius: var(--way-form-element-border-radius-medium);
```

Comment:
**suggestion**: add docblock comments for these Custom Properties. See the way-button.scss. Stencil generates a nice markdown doc from this when you run npm run build (see the /way-button/readme.me). Which we then can easily copy-paste to our doc site.